context/version.ts

```diff
@@ -48,17 +75,18 @@ const VERSION_HASH: { [k: `${typeHash}:${codeHash}`]: version } = {
   [HASHES["0.0.9"]!]: "0.0.9",
   [HASHES["0.0.10"]!]: "0.0.10",
   [HASHES["0.0.11"]!]: "0.0.11",
-  //@ts-ignore
+  //@ts-ignore  This version only exist on ghostnet
```

Comment:
```suggestion
  //@ts-expect-error - This version only exist on ghostnet
```
Just thought about it, but this way we can detect regression